src/ci/docker/host-x86_64/mingw-check-1/Dockerfile

```diff
@@ -55,6 +55,4 @@ ENV SCRIPT \
            /scripts/validate-error-codes.sh && \
            reuse --include-submodules lint && \
            python3 ../x.py test collect-license-metadata && \
-           # Runs checks to ensure that there are no issues in our JS code.
-           es-check es2019 ../src/librustdoc/html/static/js/*.js && \
-           tsc --project ../src/librustdoc/html/static/js/tsconfig.json
+	   src/tools/tidy --extra-checks=js
```

Comment:
Since this is now part of `tidy`, I think that we can move it out of this CI runner and just put it into `mingw-check-tidy`, to have centralized execution of all tidy checks on one place.